src/solvers/dg.jl

```diff
@@ -142,6 +142,34 @@ function Base.show(io::IO, ::MIME"text/plain", integral::VolumeIntegralPureLGLFi
 end
 
 
+struct VolumeIntegralStaggeredGrid{VolumeFluxDG, VolumeFluxFV, Indicator} <: AbstractVolumeIntegral
+  volume_flux_dg::VolumeFluxDG
+  volume_flux_fv::VolumeFluxFV
+  indicator::Indicator
+end
```

Comment:
For better consistency with other naming conventions, the specific part should come last, so maybe `VolumeIntegralShockCapturingSubcell`? But that might also depend on the midterm vision of this stuff.

CC @sloede (aka Mr. Consistency :sunglasses:)

Reply:
I like Hendrik's last name

Reply:
Well. The existing `VolumeIntegralShockCapturingHG` is also a subcell-based SC approach. Thus `VolumeIntegralShockCapturingSubcell` is at least ambiguous. Is there a more precise name that is not significantly longer?

Reply:
Since this volume integral contains a blending on a subcell level, maybe `VolumeIntegralShockCapturingSubcellBlending` would be more precise. On the other hand, that would be quite long (Too long? I don't know).

Reply:
Based on `VolumeIntegralShockCapturingHG`, are there some initials of the authors you could attach to `VolumeIntegralShockCapturing`?

Reply:
I would rather rate the ShockCapturingHG as an element-wise shock capturing. Hence, personally, Subcell would be fine. I can somewhat understand the confusion of MSL. It is indeed not super consistent, but unfortunately, it already carried over in our daily dialect about these methods.

Reply:
If one really wants to make it even longer and put an emphasize on the difference, an option could be

`VolumeIntegralShockCapturingLocalSubcell`

Reply:
I also see the HG variant as element-based approach (one factor per element). Here, the blending is controlled at a subcell level. Thus, `VolumeIntegralShockCapturingSubcell` is fine with me, too.

Reply:
@sloede Are you okay with that, or would you prefer `VolumeIntegralShockCapturingLocalSubcell`?

Reply:
`VolumeIntegralShockCapturingSubcell ` is fine by me (as suggested by Hendrik)